src/Extensions/Abstractions/Activities/ActivityExtensions.cs

```diff
@@ -129,6 +147,7 @@ public static class ActivityExtensions
 		private const string UserHashKey = "UserHash";
 		private const string HealthCheckMarkerKey = "HealthCheckMarker";
 		private const string HealthCheckMarkerValue = "true";
+		private const string PerformanceMarkerKey = "PerformanceMarker";
```

Comment:
```suggestion
		private const string PerformanceMarkerKey = "PerformanceTestMarker";
```
Maybe naming it `PerformanceTestMarker` would be more clear? (depending on the usage)